include/interfaces/non_conservative/dynamic_stokes.h

```diff
@@ -207,8 +207,8 @@ system_residual(const typename DoFHandler<dim>::active_cell_iterator &cell,
           // compute residual:
           local_residual[i] +=  (
                                   u_dot * v +
-                                  eta * scalar_product(sym_grad_u,grad_v) +
-                                  (1./eta)*p*div_v
+                                  eta * scalar_product(sym_grad_u,grad_v) -
+                                  (1./eta)*p*div_v + m * div_u
```

Comment:
*p div_v* should be  the transposed matrix of *m div_u*. They are related but not identical...



> On 12 Nov 2015, at 12:03, Luca Heltai <notifications@github.com> wrote:
> 
> In include/interfaces/non_conservative/dynamic_stokes.h <https://github.com/mathLab/pi-DoMUS/pull/56#discussion_r44644855>:
> 
> > @@ -207,8 +207,8 @@ system_residual(const typename DoFHandler<dim>::active_cell_iterator &cell,
> >            // compute residual:
> >            local_residual[i] +=  (
> >                                    u_dot * v +
> > -                                  eta * scalar_product(sym_grad_u,grad_v) +
> > -                                  (1./eta)*p*div_v
> > +                                  eta * scalar_product(sym_grad_u,grad_v) -
> > +                                  (1./eta)*p*div_v + m * div_u
> I think if you put -m * div_u they should be identical
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mathLab/pi-DoMUS/pull/56/files#r44644855>.
>